README.md

```diff
@@ -29,84 +29,68 @@
 ## Install
 
 ```bash
-npm install --save @webex/components @momentum-ui/react
+npm install --save @webex/components
 ```
 
 ## Usage
-**Webex Components** are built with [Momentum UI](https://github.com/momentum-design/momentum-ui) React components. We need to externally import all the fonts, icons, images and core *CSS* manually. There are two ways to do this:
-
-### 1. CSS
-1. Copy the `@momentum-ui/core/fonts` directory to the 'fonts' directory for your site.
-2. Copy the `@momentum-ui/core/images` directory to the 'images' directory for your site.
-3. Copy the `@momentum-ui/core/css/momentum-ui.min.css` to your styles/css directory.
-4. In the `<head>`, of your HTML reference the location of your momentum-ui.min.css:
-
-    `<link rel='stylesheet' href='path/to/momentum-ui/css/momentum-ui.min.css'>`
-
-***
-### 2. SCSS
-1. Add `$brand-font-folder` and `$icon-font-path` variables to the top of your app's Sass file.
 
+We need to externally import all the fonts, icons, images and core _CSS_ manually. Import `@webex/components/dist/webexComponents.css` into your main entry JavaScript file.
```

Comment:
We should also show the alternative way of importing the CSS with a `link` tag on their HTML file